protobuf/types/service.proto

```diff
@@ -54,7 +54,48 @@ message Service {
     string command = 5;               // Command to run the container.
     repeated string env = 9;          // Default env vars to apply to dependency on runtime.
   }
-  
+
+  // A workflow is a configuration to trigger a specific task when certains conditions of a trigger are valid.
+  message Workflow {
+
+    // Trigger is the configuration that will trigger a workflow.
+    message Trigger {
+
+      // Type of trigger available. Event to react to an event. Result to react to a result of an execution.
+      enum TriggerType {
```

Comment:
Can we rename it to `Type`? It's already in the scope of Trigger message

Reply:
done